plugins/lookup/nb_lookup.py

```diff
@@ -256,6 +256,26 @@ def get_endpoint(netbox, term):
         "webhooks": {"endpoint": netbox.extras.webhooks},
     }
 
+    major, minor, patch = [int(x, 10) for x in pynetbox.__version__.split(".")]
```

Comment:
Not a big deal, but I think you can do it like this as well: 
`major, minor, patch = map(int, pynetbox.__version__.split("."))`

Reply:
Put me down as knows enough python to be dangerous, but been around too long to keep track of all the changes..  looking at map vs. for loop, map is faster, so map wins.